### ISSUE TITLE
Can't read_frames

### ISSUE THREAD

**Issue by pabloriera** [EmbodiedCognition/py-c3d#4]
Hi, I have a problem reading frames on win 64 machine with latest version. It worked on Linux though.

This is the error. Any clue? Could be the c3d file or numpy version maybe? Thanks in advance

```
for i, (_, points, analog) in enumerate(C.read_frames()):

  File "build\bdist.win-amd64\egg\c3d.py", line 869, in read_frames
    count=n).reshape((self.point_used, 4))

ValueError: string is smaller than requested size

```

**Comment by lmjohns3** [comment]
Unfortunately I don't have a Windows machine to test on. It seems like it could be something with the C3D file; the error you're getting seems to be due to the file reporting that it contains more data than it actually contains (though there are other possible explanations for this error). Can you provide any additional info?

**Comment by AKuederle** [comment]
This seems to be resolved in #28. Please retest and open a new issue if you still run into problems!
